pkgs/development/go-packages/generic/default.nix

```diff
@@ -211,8 +211,7 @@ let
     '';
 
     preFixup = preFixup + ''
-      find $out/bin -type f -exec ${removeExpr removeReferences} '{}' + || true
-      find $out/libexec -type f -exec ${removeExpr removeReferences} '{}' + || true
+      find $out/{bin,libexec,lib} -type f 2>/dev/null | xargs -r -n 1 ${removeExpr removeReferences} || true
```

Comment:
Why `-n 1`? Depending on the amount of binaries, using multiple args could save a significant amount of processes. The original version with `-exec … {} +` used multiple args too.

Reply:
I was not aware of that. Will fix.

Reply:
done.